configs/default/GEPR-GEPRCF722BT.config

```diff
@@ -1,4 +1,4 @@
-# Betaflight / STM32F7X2 (S7X2) 4.1.7 May 28 2020 / 15:06:20 (9ba02a587) MSP API: 1.42
```

Comment:
what's the reason for changing API version? This will make users can't use older firmware anymore.

Reply:
Because they use new firmware to make the unified target and verify it, API version update itself.

Reply:
This will force every user who uses 4.1.x to upgrade to 4.2.x. Shouldn't be changed.

Reply:
Edit it back to 1.42.